src/frontend/src/common/utils/memo.ts

```diff
@@ -0,0 +1,19 @@
+import React, { PropsWithChildren, SFC } from "react";
+
+export const memo = <P extends Record<string, unknown>>(
+  Component: SFC<P>,
+  propsAreEqual?: (
+    prevProps: Readonly<PropsWithChildren<P>>,
+    nextProps: Readonly<PropsWithChildren<P>>
+  ) => boolean
+) => {
+  const memoized = React.memo(Component, propsAreEqual);
+
+  Object.defineProperty(memoized, "displayName", {
+    set(name) {
+      Component.displayName = name;
+    },
+  });
+
+  return memoized;
+};
```

Comment:
What does React.memo do that is different from this?  It seems like it isn't actually doing its job 🤯

Reply:
If you try to replace instances of the new `memo` with `React.memo` you will likely see the problem in your editor. For FC that are stored in variables, they are technically anonymous functions. For this reason, React does not assign them a `defaultName`, which causes the linter to fail. 

`defaultName` helps with a few things in debugging. Among them, providing a name to React devtools when displaying the component tree and giving a component a human-readable name in error messages. Otherwise it would just say `anonymous`. This isn't really React's fault. It's just an unfortunate side effect of FC and HOC sometimes.

https://reactjs.org/docs/react-component.html#displayname